test/parallel/test-assert-async.js

```diff
@@ -103,7 +103,7 @@ const invalidThenableFunc = () => {
   promises.push(assert.rejects(promise, {
     name: 'TypeError',
     code: 'ERR_INVALID_RETURN_VALUE',
-    // FIXME: This should use substring matching for key words, like /Promise/ and /undefined/
+    // FIXME(@JakobJingleheimer): This should match on key words, like /Promise/ and /undefined/
```

Comment:
```suggestion
    // FIXME(JakobJingleheimer): This should match on key words, like /Promise/ and /undefined/.
```
To align with other `FIXME` comments https://github.com/nodejs/node/blob/dbe5874c7e21658113993d07a2426299edfc2f8c/lib/net.js#L779

Reply:
🤷‍♂️ Sure. I checked before I updated it and found a mix (but more with the `@`)

I decided against the ending period because of the regex immediately before it (to avoid confusion).

Reply:
Ah OK, I've searched for `// FIXME(@` and got no results. Anyway that's certainly not blocking.

Reply:
Ah, I had searched for `TODO`

<img width="333" alt="search results for TODO" src="https://user-images.githubusercontent.com/3012099/175816845-af901562-0b4b-4a2a-9800-d5bbec3cbea2.png">